test/app/submissions_test.rb

```diff
@@ -43,7 +43,7 @@ def assert_response_status(expected_status)
   end
 
   def test_guests_can_view_submissions
-    Attempt.new(alice, Iteration.new('word-count/file.rb' => 'CODE')).save
+    Attempt.new(alice, Iteration.new('ruby/file.rb' => 'CODE')).save
```

Comment:
If what you're doing is creating a test for an obviously invalid slug I would not use a track id, but a slug like `invalid-slug`.